buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/SnippetsTask.groovy

```diff
@@ -226,10 +227,14 @@ class SnippetsTask extends DefaultTask {
                                 return
                             }
                             if (it.group(7) != null) {
-                                snippet.warnings.add(it.group(7))
+                                snippet.teardown = it.group(7)
```

Comment:
These are all from the bad old days where we really didn't have tests for this sort of thing.